its/ruling/src/integrationTest/java/org/sonarsource/iac/IacRulingTest.java

```diff
@@ -48,13 +49,17 @@ class IacRulingTest {
   private static final String LITS_VERSION = "0.11.0.2659";
   private static final File LITS_OUTPUT_DIRECTORY = FileLocation.of("build/reports/lits").getFile();
   private static final File LITS_DIFFERENCES_FILE = Path.of(LITS_OUTPUT_DIRECTORY.toURI()).resolve("differences").toFile();
+  private static final String JAVA_VERSION = "7.34.0.35958";
   private static final String SCANNER_VERSION = "5.0.1.3006";
+  @TempDir
+  private static Path temporaryDirectory;
 
   @RegisterExtension
   static OrchestratorExtension orchestrator = OrchestratorExtension.builderEnv()
     .useDefaultAdminCredentialsForBuilds(true)
     .setSonarVersion(System.getProperty(SQ_VERSION_PROPERTY, DEFAULT_SQ_VERSION))
     .addPlugin(FileLocation.byWildcardFilename(new File("../../sonar-iac-plugin/build/libs"), "sonar-iac-plugin-*-all.jar"))
+    .addPlugin(MavenLocation.of("org.sonarsource.java", "sonar-java-plugin", JAVA_VERSION))
```

Comment:
Did you also test manually in SQ that it's working? If you got the time, I think we should do it.

Reply:
Yes, I verified that it's working and even on a project with properties files nothing crashes.